### PR TITLE
feat(android): polygon bounds

### PR DESCRIPTION
**Pull request by m1ga**
Add `bounds` property to `Polygon`. Will return the bounding box of the polygon. 

Example:

```js
const Map = require('ti.map');
const win = Ti.UI.createWindow();
const mapView = Map.createView({
	region: {
		latitude: 7.338208,
		longitude: 1.886329,
		latitudeDelta: 0.1,
		longitudeDelta: 0.1
	}
});

const polygon = Map.createPolygon({
	name: 'polygon_object',
	strokeWidth: 3,
	strokeColor: '#ffff00',
	fillColor: '#ff00ff',
	points: [{
			latitude: 53.677150,
			longitude: -7.728051
		},
		{
			latitude: 60.860020,
			longitude: 8.667197
		},
		{
			latitude: 50.396644,
			longitude: 9.769200
		},
		{
			latitude: 47.657334,
			longitude: -0.213136
		}
	]
});

win.add(mapView);
mapView.addPolygon(polygon);
setTimeout(function() {
	console.log(polygon.bounds);
	mapView.region = polygon.bounds;
}, 2000)
win.open();
```

will fit the polygon into the view.

[ti.map-android-5.6.0.zip](https://github.com/tidev/ti.map/files/12707159/ti.map-android-5.6.0.zip)

### CODE REVIEW

**Comment by hansemannn** [comment]
@m1ga Can you bump the version, change the "since" version to 12.3.0 (as ti.map currently uses the SDK version as the base -- I know, horrible, but that requires a whole change once)? Also, if possible, attach the new binary.

File: apidoc/Polygon.yml [hunk 1]
```diff
@@ -57,3 +57,9 @@ properties:
     availability: creation
     type: Number
     platforms: [android]
+
+  - name: bounds
+    summary: Returns the bounding box of the polygon. Useful to center the region.
+    type: MapRegionTypev2
+    platforms: [android]
+    since: "5.5.0"
```
Comment (m1ga) [inline apidoc/Polygon.yml:65]:
```suggestion
    since: "12.3.0"
```

File: android/manifest [hunk 2]
```diff
@@ -2,7 +2,8 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 5.5.3
+
```
Comment (m1ga) [inline android/manifest:5]:
```suggestion
```

**Comment by m1ga** [comment]
sorry for the multiple updates, not at my main machine right now but I've made the changes in githubs editor and build a new version (see original post)

**Pull request merged**